test/models/default-filters-test.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert')
+const ElectronConfig = require('electron-config')
+const storage = new ElectronConfig({ name: 'config-test' })
+const DefaultFilters = require('../../src/models/DefaultFilters')
+const login = 'RupertScrivener'
+
+describe('DefaultFilters', () => {
+  describe('init', () => {
+    xit('calls each filter function and then the save function', () => {
+      // what is the preferred way to spy on functions?
```

Comment:
Maybe instead of spying on what the function does, assert its side effects? Assert there's no default-filters key in storage, call init, then check that `storage.has('default-filters')` is true. Could also assert that `filters` in storage now has 'My Issues', etc. in it.